test/sequential/test-util-debug-monkeypatched.js

```diff
@@ -0,0 +1,92 @@
+// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
Since most of the test code here is copied nearly verbatim from another test, I think keeping that test's license is appropriate. I added a few lines, but that's it. I made it a separate test to avoid having to wrestle with side effects from this test affecting other tests.